nipype/interfaces/freesurfer/model.py

```diff
@@ -1144,7 +1144,7 @@ class Label2VolInputSpec(FSTraitedSpec):
     invert_mtx = traits.Bool(
         argstr='--invertmtx', desc='Invert the registration matrix')
     fill_thresh = traits.Range(
-        0., 1., argstr='--fillthresh %.f', desc='thresh : between 0 and 1')
+        0., 1., argstr='--fillthresh %f', desc='thresh : between 0 and 1')
```

Comment:
Let's go ahead and change this to `%g` (and remake auto tests).

Also, in the Label2Vol docstring, you'll need to change `--fillthresh 0` to `--fillthresh 0.5`.